pkg/cmd/pr/shared/finder.go

```diff
@@ -211,6 +220,19 @@ func (f *finder) Find(opts FindOptions) (*api.PullRequest, ghrepo.Interface, err
 		fields.Remove("projectItems")
 	}
 
+	// TODO projectsV1Deprecation
+	// Remove this block
+	if fields.Contains("projectCards") {
+		if opts.Detector == nil {
+			cachedClient := api.NewCachedHTTPClient(httpClient, time.Hour*24)
+			opts.Detector = fd.NewDetector(cachedClient, f.baseRefRepo.RepoHost())
+		}
+
+		if opts.Detector.ProjectsV1() == gh.ProjectsV1Unsupported {
```

Comment:
I would have preferred to take the same approach as the `issue` commands and to have made the decision to add this field in `pr view`, but I really, really didn't want to untangle the PR Finder like I did with the issues lookup work. Furthermore, the `issues` work was a lot more viral with a `Detector` needing to be passed as an arg in many many places, where this is a lot more self contained due to the finder options field being optional.

Reply:
Did a spike today to explore the other option and it actually ended up adding a lot of extra stuff to `pr view` that would need to be removed, so think just leaving it as is is best.